source/documentation/get-all-messages.html.md.erb

```diff
@@ -5,25 +5,54 @@ weight: 60
 
 # Get all messages for an excise movement
 
-This is draft information for the prototype and is likely to change.
+This is prototype information and is likely to change.
 
 Use the GET endpoint [Get all messages for an excise movement using the movement ID](/api-documentation/docs/api/service/excise-movement-control-system-api/1.0/oas/page#tag/Movements/operation/getAllMessages) to get all messages for a specific excise movement.
 
-## Filters
+## Requirements 
+
+The consignor ID must match an excise reference number (ERN) in the bearer token that you use for API authentication. If it does not, a 403 Forbidden error will be thrown.
+
+## Query parameters
+
+| Parameter | Type | Details |
+| -- | -- | -- |
+| Date range | String | Filter messages by date range |
+| Message type| String | Filter messages by message type |
+| LRN | String | Filter messages by local reference number (LRN)|
+
+## Messages
+
+You can receive the following message types from this endpoint:
+
+| Message | Schema link |
+| --- | --- |
+| Generic refusal message (IE704) | [XSD](https://github.com/hmrc/excise-movement-control-system-api/blob/main/app/xsd/ie704uk.xsd) |
+| E-AD/E-SAD container (IE801) | [XSD](https://github.com/hmrc/excise-movement-control-system-api/blob/main/app/xsd/ie801.xsd) |
+| Reminder Message for Excise Movement (IE802) | [XSD](https://github.com/hmrc/excise-movement-control-system-api/blob/main/app/xsd/ie802.xsd) |
+| Notification of Diverted E-AD/E-SAD (IE803) | [XSD](https://github.com/hmrc/excise-movement-control-system-api/blob/main/app/xsd/ie803.xsd) |
+| Interruption of Movement (IE807) | [XSD](https://github.com/hmrc/excise-movement-control-system-api/blob/main/app/xsd/ie807.xsd) |
+| Cancellation (IE810) | [XSD](https://github.com/hmrc/excise-movement-control-system-api/blob/main/app/xsd/ie810-v16-0.xsd) |
+| Change of destination (IE813) | [XSD](https://github.com/hmrc/excise-movement-control-system-api/blob/main/app/xsd/ie813-v16-0.xsd) |
+| Submit a new draft movement (IE815) | [XSD](https://github.com/hmrc/excise-movement-control-system-api/blob/main/app/xsd/ie815-v16-0.xsd) |
```

Comment:
The XML schema we have for Get New Messages from EMCS Core doesn't have IE815 on it as a valid response type?

Reply:
Correct, removing.